ftw/theming/extensions.py

```diff
@@ -53,15 +53,21 @@ def find_relative_resource(relpath):
     return path
 
 
+def chunks(l, n):
+    if not l:
+        return []
+    return [l[i:i + n] for i in xrange(0, len(l), n)]
+
+
 def fill_svg(data, fill_css=None, fill_xpath=None):
     doc = etree.fromstring(data)
     expressions = []
 
-    for xpr, color in fill_xpath or ():
+    for xpr, color in chunks(fill_xpath, 2) or ():
```

Comment:
As far as I understand it, the data structure of `fill_xpath` and `fill_css` changes. You ma need to inspect the data structure / types in order to decide whether its the new or the old style. Then you may convert the one into the other so that you then only work with one data structure style.

Please test both.